src/app/legacy/containers/Header/index.jsx

```diff
@@ -79,6 +81,8 @@ const HeaderContainer = ({
     }
   }
 
+  const renderLiteSiteCTA = isLite && liteEnabledServices.includes(service);
```

Comment:
I will flag to Harry, though this banner is required for UX and A11y on the Lite site so I don't think we would want to give editors the ability to turn it off.

Reply:
Ah yes, that's a good point. We have certainly used toggles in the past to roll things out, rather than for something to be switched off. Once it has been rolled out to all services, the toggle logic could then be removed. Let's not hold up getting the PR merged, we can always revisit this later.